src/sage/sets/disjoint_set.pyx

```diff
@@ -6,26 +6,25 @@ The main entry point is :func:`DisjointSet` which chooses the appropriate
 type to return. For more on the data structure, see :func:`DisjointSet`.
 
 This module defines a class for mutable partitioning of a set, which
-cannot be used as a key of a dictionary, vertex of a graph etc. For
+cannot be used as a key of a dictionary, a vertex of a graph, etc. For
 immutable partitioning see :class:`SetPartition`.
 
 AUTHORS:
 
 - Sébastien Labbé (2008) - Initial version.
 - Sébastien Labbé (2009-11-24) - Pickling support
 - Sébastien Labbé (2010-01) - Inclusion into sage (:issue:`6775`).
+- Giorgos Mousa (2024-04-22): Optimize
 
 EXAMPLES:
 
 Disjoint set of integers from ``0`` to ``n - 1``::
 
-    sage: s = DisjointSet(6)
-    sage: s
+    sage: s = DisjointSet(6); s
```

Comment:
It is better to avoid semi-colon to improve readability [see E702](https://www.flake8rules.com/rules/E702.html).

Reply:
Not so elegant, but ok.